src/fastoad/notebooks/02_explanation/02_pure_python.ipynb

```diff
@@ -0,0 +1,836 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    print("MTOW input: ", np.round(MTOW_result, 1), "kg")
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Changed in dfe233f7